configs/module-defaults.config

```diff
@@ -127,6 +127,22 @@ params {
             publish_dir      = "tantan_to_GFF3"
             publish_results  = "all"
         }
+        'star_genomeGenerate' {
+            args             = ""
+            publish_dir      = "star_genomeGenerate"
+            publish_results  = "all"
+            sjdbGTFfile          = ""
+            sjdbFileChrStartEnd  = ""
+            varVCFfile           = ""
+        } 
+        'star_alignReads' {
+            args                 = ""
+            publish_dir          = "star_alignReads"
+            publish_results      = "all"
+            sjdbGTFfile          = ""
+            sjdbFileChrStartEnd  = ""
+            varVCFfile           = ""
+        }    
```

Comment:
Can you remove camel case to keep the naming convention? `star_genome_generate`

Reply:
Thank you Chris, great points! I'll combine the tests then and remove camel case.